Userland/Libraries/LibC/CMakeLists.txt

```diff
@@ -269,6 +274,10 @@ foreach(RELATIVE_HEADER_PATH IN LISTS HEADERS)
     file(CREATE_LINK "${CMAKE_CURRENT_SOURCE_DIR}/${RELATIVE_HEADER_PATH}" "${CMAKE_STAGING_PREFIX}/${CMAKE_INSTALL_INCLUDEDIR}/${subdirectory}/${name}" SYMBOLIC)
 endforeach()
 
+if (LIBC_LINK_HEADERS_ONLY)
+    return()
```

Comment:
My previous comment implies 

1) moving the link creation to a function in a secondary file in this directory
2) moving the header file list into that file
3) having the signature take the path to libc source dir as an argument instead of relying on CMAKE_CURRENT_SOURCE_DIR.
4) calling that function here